src/main/java/stringcalculator/domain/Experssion.java

```diff
@@ -50,33 +66,21 @@ private void validateOperator(String term) {
     }
 
     public int reduce() {
+        createOperatorMap();
+
         int result = operands.get(0).intValue();
 
         for (int i = 0; i < operators.size(); i++) {
-            result = calculate(result, operands.get(i + 1).intValue(), operators.get(i));
+            result = operatorMap.get(operators.get(i)).calculate(result, operands.get(i + 1).intValue());
```

Comment:
```suggestion
            Operator operator = operatorMap.get(operators.get(i));
            result = operator.calculate(result, operands.get(i + 1));
```
이렇게 나눠 주는게 좀 더 가독성에 좋겠네요 :)

Reply:
매소드 채이닝이 아니라 이렇게 한번 끊고 가야 하는건가요? 
이게 가독성에 도움이 된다는 이유는 무언가요? 길이에 대한 문제일까요?